src/bots/telegram/client.js

```diff
@@ -213,6 +215,8 @@ function prepareMessage(msg: Telegram$Message): string {
   if (msg.forward_from) {
     const name: string = Name.from(msg.forward_from);
 
+    text = prepareQuote(text);
```

Comment:
forwarded message should be fully displayed, as in the telegram

Reply:
i cut it off from here